integration/base/pom.xml

```diff
@@ -113,13 +113,20 @@
             <scope>provided</scope>
             <exclusions>
                 <exclusion>
+        <!-- transitive dependency via wiremock -->
+        <!-- when wiremock is updated beyond 2.35, this can be removed -->
```

Comment:
This line refers to json-smart -- can you move it back to right before the json-smart dependency?